test/utils/DataStoreStub.ts

```diff
@@ -0,0 +1,20 @@
+import { Account } from 'aws-sdk/clients/organizations';
+import { SSMDataStore } from '../../src/DataStore';
+import { destructureAccount } from '../../src/utils';
+import { unionBy } from 'lodash';
+
+export const dataStore = new SSMDataStore();
```

Comment:
It would be more readable to extract a `DataStore` interface from `SSMDatastore`, and have a separate stub implementation:

```
            DataStore
               ^
               |
     ---------------------------
     |                         |
SSMDataStore       InMemoryDataStore

```